pandas/io/excel.py

```diff
@@ -68,40 +67,43 @@
     * 1 -> 2nd sheet as a DataFrame
     * "Sheet1" -> 1st sheet as a DataFrame
```

Comment:
```suggestion
    - ``"Sheet1"``: Load sheet with name "Sheet1"
```
And equivalent in the rest.